core-developers/become-core-developer.rst

```diff
@@ -59,6 +60,41 @@ are granted through these steps:
      were in the form of a separate post on the already open topic with
      the poll.
 
+Poll template
+=============
+
+.. _coredev-template:
+
+While Discourse uses Markdown for formatting, the poll functionality is
+custom and somewhat resembles BBcode. There's a creator for polls in the
+UI (click the Cog icon in the edit box toolbar and choose "Build Poll").
+Make sure to choose "Show Results... When the poll is closed".
+
+Alternatively, here is a template for a poll that you can copy:
+
+.. code-block:: bbcode
+
+   [poll type=regular results=on_close public=false chartType=bar groups=committers close=2024-07-15T21:15:00.000Z]
+   * Promote Basil Fawlty
+   * Do not promote
+   [/poll]
+
+.. raw:: html
+
+    <script>
+    for (let span of document.querySelectorAll('span')) {
+      console.log(span.textContent);
+      if (span.textContent === '2024-07-15T21:15:00.000Z') {
+         const nextWeek= new Date();
+         nextWeek.setDate(nextWeek.getDate() + 7);
+         nextWeek.setMinutes(0);
```

Comment:
Shall we leave minutes intact? If you copy this just before the new hour (like 8:58), we lose nearly an hour of voting. 
```suggestion
```